src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -67,7 +64,7 @@ const ProjectActions = ({
       <FormattedMessage
         defaultMessage="Done"
         description="Generic success message displayed when a list is updated or deleted"
-        id="projectActions.savedSuccessfully"
+        id="savedSearchActions.savedSuccessfully"
```

Comment:
Please let's keep existring string ids as they were to avoid unnecessary re-translation work and costs.

Reply:
Done